### ISSUE TITLE
[Bug] Couldn't Sync

### ISSUE THREAD

**Issue by Tahinli** [AppFlowy-IO/AppFlowy#6448]
### Bug Description

Yesterday I was writing a document that includes code blocks and tables. It wasn't syncing properly. Whenever I want to export with html, markdown or even with publishing to website, it showed less then what I wrote. I also checked in my mobile and it was not fully synced there. I tried to closing and repopening sync feature in settings but it didn't solve. When I closed my document and opened again, I lost some of my writings and code blocks. I don't know why it happening but I just solved the problem by:

Copying text to somewhere else
close the document
reopen and reorganize and repaste it,
repeat until sync everything properly

### How to Reproduce

I don't know what triggers the problem.

### Expected Behavior

Syncing properly

### Operating System

Fedora 40

### AppFlowy Version(s)

0.7.0

### Additional Context

I think this must be considered 
https://github.com/AppFlowy-IO/AppFlowy/issues/6192

Also there can be a button to trigger sync manually like force push.

**Comment by LucasXu0** [comment]
@Tahinli Could you attach your log files or send them to my [email](mailto:lucas.xu@appflowy.io)? Here's the guide on how to export them.

https://appflowy.com/ccf3ca5a-7ecd-4c35-805c-3a3c9c22e87d/AppFlowy-Q-A-9eeac747-8652-4849-b23f-c631504dcbee

**Comment by Tahinli** [comment]
I think this is the right day :D

[log.2024-10-01.txt](https://github.com/user-attachments/files/17250992/log.2024-10-01.txt)

**Comment by Tahinli** [comment]
This situation continuous on my mobile too. There is some problem that breaks my experience a lot.

**Comment by 0x00000oak** [comment]
I like the app very much but I also have similar problems and it prevents us from using it effectively. Trying to collaborate with other users only seems to work when opening the same note at the same time. We can even see each others cursors which is nice but that not the most common use case for us at least. There should at least be a manual sync button in the client apps but ideally the detection and syncinf of changes should be improved.

**Comment by khorshuheng** [comment]
> I like the app very much but I also have similar problems and it prevents us from using it effectively. Trying to collaborate with other users only seems to work when opening the same note at the same time. We can even see each others cursors which is nice but that not the most common use case for us at least. There should at least be a manual sync button in the client apps but ideally the detection and syncinf of changes should be improved.

We have deployed a fix last week, in which if a user open the document that was edited by another user which is no longer online, the document should be updated within a few seconds. Is this still happening for your case? And if so, could you give a brief description that allows us to reproduce the issue consistently?

**Comment by 0x00000oak** [comment]
Hi thank you for the quick answer I am using the self hosted with iOS version `0.7.6`. Does the server version also matter regarding the fix?

**Comment by khorshuheng** [comment]
Yes, the server version matters, because we fixed two bugs related to synchronization in version 0.8.2 and 0.8.6 of AppFlowy Cloud recently.

**Comment by 0x00000oak** [comment]
Thank you I will check what version I have currently and update/re-test.
